manifests/init.pp

```diff
@@ -133,8 +133,8 @@
 
     $normalized_repos.each |$yumrepo, $attributes| {
       if member($_managed_repos_minus_exclusions, $yumrepo) {
-        Resource['yumrepo'] {
-          $yumrepo: * => $attributes,
+        yumrepo { $yumrepo:
```

Comment:
This isn't quite identical behaviour.  In the original form, I suspect there isn't a duplicate resource situation if the yumrepo has already been declared.  On the other hand, I don't think this is actually the case here looking through https://github.com/voxpupuli/puppet-yum/commit/1fd551cdcac27d6f1389b7ee3f86af6cfa664c3c